llvm/lib/Transforms/Utils/CloneFunction.cpp

```diff
@@ -135,6 +135,29 @@ void llvm::CloneFunctionAttributesInto(Function *NewFunc,
                          OldAttrs.getRetAttrs(), NewArgAttrs));
 }
 
+DISubprogram *llvm::ProcessSubprogramAttachment(const Function &F,
```

Comment:
Could you add documentation for this function? The term "Process" is not very descriptive. IIUC this function is visiting instructions / SPs  and collecting debug information?